subprojects/gradle/instrumentation-tests/src/main/kotlin/com/avito/instrumentation/InstrumentationTestsTask.kt

```diff
@@ -87,6 +87,10 @@ abstract class InstrumentationTestsTask @Inject constructor(
     @Optional
     val suppressFailure = objects.property<Boolean>()
 
+    @Input
+    @Optional
```

Comment:
why it is optional? we can always use default here

Reply:
Discussed. Right now we won't change the behavior of the optional properties of this task.